_component_design/cards.md

```diff
@@ -252,18 +181,18 @@ If an individual card or set of cards is loading, use the Spinner Loading Indica
 {% endcolumn %}
 </div>
 
+</section>
+
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
-
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
 ### Empty card
 
-When the card grid is customizable by the user the Empty Card is where the user can add a new card.
-
-The empty card button is based on the [chip] component and is created by removing the fill of a chip, adding a dotted stroke to the containing shape, and overlaying a helix plus icon
+The empty card provides a space for a user to add a new card.
 
+The empty card button is based on the [chip](http://helix.rax.io/components/chips.html) component and is created by removing the fill of a chip, adding a dotted stroke to the containing shape, and overlaying a Helix plus icon (+).
```

Comment:
Some components are capitalized (e.g., the Helix Grid) and some are not (e.g., chip). I'd go one way or the other.